docs/Spark/gems/source-target/file/iceberg.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Iceberg
+id: iceberg
+description: Iceberg
+sidebar_position: 6
+tags:
+  - gems
+  - file
+  - iceberg
+---
+
+Reads and writes Iceberg tables, including Iceberg Merge operations and Time travel.
+
+## Required Settings
+
+Before you can use Iceberg source Gems, you must configure some required settings at the environment, initialization, and runtime stages.
+
+### Environment Setting
+
+You must configure a required Spark JAR dependency in your Fabric environment.
```

Comment:
Would it help to include where users can configure this?

Reply:
Yes, we should add that or can be covered in demo video.

Reply:
@alexanderahn - Those Jar dependencies should be available on their compute platform wherever Spark is installed like EMR, Dataproc, Syanpse, Databricks Cluster etc.